android-lib/src/androidTest/java/com/kinvey/androidTest/store/DataStoreTest.java

```diff
@@ -2183,6 +2183,22 @@ public void testSaveList() throws InterruptedException {
         assertNotNull(pushCallback.getResult());
     }
 
+    @Test
+    public void testSaveListWithStoreTypeCache() throws InterruptedException {
+        client.enableDebugLogging();
+        TestManager<Person> testManager = new TestManager<>();
+        testManager.login(TestManager.USERNAME, TestManager.PASSWORD, client);
+        DataStore<Person> store = DataStore.collection(Person.COLLECTION, Person.class, StoreType.CACHE, client);
+        ArrayList<Person> persons = new ArrayList<>();
+        persons.add(new Person(TEST_USERNAME));
+        persons.add(new Person(TEST_USERNAME_2));
+        persons.add(new Person(TEST_TEMP_USERNAME));
+
+        CustomKinveyListCallback<Person> saveCallback = testManager.saveCustomList(store, persons);
+        assertNotNull(saveCallback.getResult());
+        assertEquals(3, saveCallback.getResult().size());
+    }
```

Comment:
Can an assert be added to check that the size of the cache matches the expected result?